src/quo2/components/drawers/bottom_actions/view.cljs

```diff
@@ -0,0 +1,53 @@
+(ns quo2.components.drawers.bottom-actions.view
+  (:require [quo2.components.buttons.button.view :as button]
+            [quo2.components.drawers.bottom-actions.style :as style]
+            [quo2.theme :as quo.theme]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+(def default-props
+  {:button-one-type :primary
+   :button-two-type :grey
+  })
```

Comment:
Please move the trailing bracket to the above line and leave one line space between each method.

```suggestion
(def default-props
  {:button-one-type :primary
   :button-two-type :grey})
   
```